src/app/containers/SubHeading/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
This whole file is essentially identical to the Heading container. It needs refactoring here.